src/framework/ui/button/button.component.tsx

```diff
@@ -84,7 +84,9 @@ export class Button extends React.Component<Props> {
       ...containerParameters
     } = style;
 
-    const alignment: ButtonAlignment = ButtonAlignments.parse(this.props.alignment, ALIGNMENT_DEFAULT);
+    const { iconAlignment } = this.props;
+
+    const alignment: ButtonAlignment = ButtonAlignments.parse(iconAlignment, ALIGNMENT_DEFAULT);
```

Comment:
* Rename type to `ButtonIconAlignment`
* Rename `ButtonAlignments`

Reply:
Done